src/sale.js

```diff
@@ -5,18 +5,61 @@ const list = document.getElementById('product');
 const showAll = document.getElementById('all');
 const bathBomb = document.getElementById('bath-bomb');
 const soap = document.getElementById('soap');
-const sets = document.getElementById('soap sets');
-const stuff = document.getElementsByClassName('bath bomb');
+const sets = document.getElementById('sets');
 
 
-for(let i = 0; i < products.length; i++) {
-    const banana = products[i];
-    const dom = rendersProducts(banana);
-    list.appendChild(dom);
-    dom.classList.add('hidden');
-}
-
 showAll.addEventListener('click', () => {
```

Comment:
You'll want to make it so that all of the products show on page landing. As it is right now, you don't see any products until you click the See All button.

Reply:
Yeah, I forgot to branch and didn't realize until over halfway through.  I'll make sure to check that more often!  Will make change so it loads right away and not onclick.